src/vivarium_nih_us_cvd/constants/results.py

```diff
@@ -64,6 +67,7 @@
     "sbp_medication_person_time": SBP_MEDICATION_PERSON_TIME_TEMPLATE,
     "ldlc_medication_person_time": LDLC_MEDICATION_PERSON_TIME_TEMPLATE,
     "outreach_intervention_person_time": OUTREACH_INTERVENTION_PERSON_TIME_TEMPLATE,
+    "polypill_intervention_person_time": POLYPILL_INTERVENTION_PERSON_TIME_TEMPLATE,
```

Comment:
You can get away with combining the outreach and polypill templates

Reply:
Jeez, I just love making things harder for myself don't I? Done.